<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkCameras</class>
 <widget class="QDialog" name="checkCameras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>721</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera Settings</string>
   </property>
   <widget class="QLabel" name="camera_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="camera_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>..</string>
    </property>
   </widget>
   <widget class="QLabel" name="camera_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Scanner Setup</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Scanner ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="scannerCode"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera Setup</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Camera ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="cameraCode"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="camLabel_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="camLabel_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="camLabel_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okBtn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>In the bellow images you can find the camera IDs.
Assign an ID to Scanner and Camera fields.</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>camera_2</zorder>
  <zorder>camLabel_3</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>checkCameras</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
